src/main/resources/reference.conf

```diff
@@ -277,7 +277,7 @@ target {
     chembl = ${drug.chembl-target}
     chemical-probes = {
       format = "json"
-      path = ${common.input}"/target-inputs/chemicalprobes/chemicalprobes.json"
+      path = ${common.input}"/target-inputs/chemicalprobes/chemicalprobes.json.gz"
```

Comment:
I think we’ve gone through this change in the past

Reply:
The latest PIS version says the the probes output is `.json`. The input definitely should come gzipped.

Reply:
Yes, I remember changing it from one to the other or from the other to the one a very long time ago, I was just wondering why that change didn't make it to the _reference.conf_ file

Reply:
It was just curiosity, no worries.

Thanks @ireneisdoomed